src/components/common/searchbar/index.js

```diff
@@ -31,7 +31,7 @@ function SearchBar({ searchVisible, setData }) {
       <div className="searchOuter">
         <div className="searchBtns"></div>
         <Search
-          defaultValue="Search through your inventory"
+          placeholder="Search through your inventory"
```

Comment:
its anoying to have to take all the text on the field to type yours placeholder was a really good call here